server/tools/README.md

```diff
@@ -0,0 +1,131 @@
+# Encryption Tools
```

Comment:
Please reduce duplication. Copy-pasting saves time short-term but creates problems in the long run. It requires multiple documents to be updated for every change.